websites/K/KissAnime/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.15",
+  "apiVersion": 1,
+  "author": {
+    "id": "432215088686956565",
+    "name": "_xdoritos"
+  },
+  "service": "KissAnime",
+  "description": {
+    "en": "KissAnime is a popular online platform offering a vast collection of anime series and movies, available for free streaming in various video qualities. Perfect for anime fans looking to watch their favorite titles easily and quickly."
+  },
+  "url": [
+    "kissanime.com.ru",
+    "www.kissanime.com.ru"
+  ],
+  "version": "1.0.0",
```

Comment:
Please add a `regExp` field. This will replace the functionality of the `url` field in the future.